gradle.properties

```diff
@@ -1,2 +1,4 @@
 version=5.0-SNAPSHOT
 publishUrl=file:../marklogic-data-hub/releases
+systemProp.file.encoding=utf-8
+org.gradle.jvmargs='-Dfile.encoding=UTF-8'
```

Comment:
Are these changes needed for handling the URI with diacritics ?

Reply:
Not exactly. With this (and no other code changes), our gradle plugin will start handling them, because we're forcing the build to happen in UTF8 - though in general we should be always adhering to UTF-8 in our builds because that's what marklogic server uses by default.